tests/test_claim_release_entrypoint_wiring.py

```
@@ -0,0 +1,32 @@
+
+    # Register the original factory with monkeypatch so teardown restores it
+    # even though entrypoint.main() assigns the production class directly.
+    original_factory = claim_release_entrypoint.bot.OpenAILegalService
+    monkeypatch.setattr(claim_release_entrypoint.bot, "OpenAILegalService", original_factory)
+    monkeypatch.setattr(
+        claim_release_entrypoint.claim_quality_hotfix,
+        "install_runtime_hotfix",
+        lambda: seen.setdefault("hotfix", True),
+    )
+
+    async def fake_main():
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `fake_main`

Add return type annotation: `None`

(ANN202)